app/src/main/res/layout/fragment_more.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is huge, try to break it down into smaller views and use `merge` technique and include those views here.